### PR TITLE
chore(release): version bump and update changelog for 5.1

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

**Comment by juliemr** [comment]
Please change the commit title to include "for 5.1" for ease of future searches

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,199 @@
...
+Webdriver-manager will now by default grab the latest versions of all binaries
+(standalone, chromedriver, iedriver, gecko driver). Use the
+`--versions.(binary name)` to pin to a specific version. Selenium standalone
+3.0.1 has a bug which prevents it from working with any version of FireFox.
+We have tested version 3.0.0-beta4 and know that it works with
+FireFox 51, and we expect that the 3.0.2 release will also work.
+
+#### Breaking changes summary:
+
+- `browser.getLocationAbsUrl` deprecated
```
Comment (juliemr) [inline CHANGELOG.md]: Clarify that this was deprecated, and has now been removed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,199 @@
...
+- Replaced `browser.rootEl` with `browser.angularAppRoot()`
```
Comment (juliemr) [inline CHANGELOG.md]: Note that `browser.rootEl` will still work - so I'd prefer to not list this here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,199 @@
...
+
+## Bug Fixes
```
Comment (juliemr) [inline CHANGELOG.md]: Please move the feature section up above the bug fixes session.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,199 @@
...
+
+  This adds two options, both of which are implemented with Blocking
+  Proxy. --webDriverLogDir will create a readable log with timing
+  information of webdriver commands in the specified directory.
+
+  --highlightDelay will pause before clicking on elements or sending keys.
+  While paused, the element that's about to be affected will be
+  highlighted.
+
+- ([8d2fc07](https://github.com/angular/protractor/commit/8d2fc07ed28a1b19c03a9869442f76f2963e40a1))
```
Comment (juliemr) [inline CHANGELOG.md]: Just move this whole thing into the breaking changes section and out of here.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,199 @@
...
+  - iedriver 3.0.0
+
+   A note on FireFox support: Selenium standalone 3.0.1 has a bug which prevents it from working
+  with any version of FireFox. We have tested version 3.0.0-beta4 and know that it works with
+  FireFox 51, and we expect that the 3.0.2 release will also work.
+
+   closes #4033
+
+- ([cd084a0](https://github.com/angular/protractor/commit/cd084a0ca29cd73aa3ce1650188adf7ddfdb7962))
+  deps(jasmine): update jasmine to ^2.5.3 (#3960)
```
Comment (juliemr) [inline CHANGELOG.md]: Is this where we should talk about `@types/jasminewd`?

File: CHANGELOG.md (hunk 3 above)
Comment (cnishina) [inline CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (juliemr) [inline CHANGELOG.md]: Sorry, my bad, it has NOT been removed just deprecated.

File: CHANGELOG.md (hunk 4 above)
Comment (juliemr) [inline CHANGELOG.md]: Nevermind, ignore, either keep it here or move to fixes.

File: CHANGELOG.md (hunk 1 above)
Comment (cnishina) [inline CHANGELOG.md]: Removed "breaking change" since it is more of a "fix"

File: CHANGELOG.md (hunk 2 above)
Comment (cnishina) [inline CHANGELOG.md]: Removed section so ignore this.

File: CHANGELOG.md (hunk 4 above)
Comment (cnishina) [inline CHANGELOG.md]: Moving this to the "fix" section

File: CHANGELOG.md (hunk 5 above)
Comment (cnishina) [inline CHANGELOG.md]: Added a section at the top to highlight this.

**Comment by juliemr** [comment]
LGTM

**Pull request merged**